ec-europa/europa-component-library
# fix(template): missing variable in if-logic - no issue

Missing variable in the "is not" logic causes the footer twig template to break (at least in twig-2.4.3/PHP)

# PR description

Bugfix for template

## QA Checklist

Commit done from my smartphone (GH down), no particular QA has been done

In order to ensure a safe and quick review, please check that your PR follow those guidelines:
